digitaltwins/device/src/digital_twin_client.ts

```diff
@@ -115,8 +116,8 @@ export class DigitalTwinClient {
   //
   private _components: {[key: string]: ComponentInformation} = {};
   //
-  // Each Digital Twin client can have only one capability model associated with it.
-  // The dcm is in DTMI format.
+  // Each Digital Twin client can have only one Model ID associated with it.
+  // The ModelID is in DTMI format.
```

Comment:
This comment is simply isolated.  Think it was for the dcm we stored here.  It's no longer at this location.